packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerPlugin.java

```diff
@@ -113,6 +116,13 @@ public void onImageReady(List<Image> images) {
               }
             });
         return true;
+      } else if (resultCode == Activity.RESULT_CANCELED) {
```

Comment:
You could just do this
```
} else if (resultCode != Activity.RESULT_CANCELED) {
       pendingResult.error("PICK_ERROR", "Error taking photo", null);
}
pendingResult = null;
methodCall = null;
```

Reply:
Good point. Thanks for the comment @Alexays .  I updated the code and can you please review again?